### DISCUSSION TITLE
Support for non-Docker use on MacOS and Windows?

### DISCUSSION THREAD

**Discussion by spikecodes** [tensorchord/pgvecto.rs#542] [Q&A] [answered]
Is support for this extension on MacOS and Windows installations of PostgreSQL available? Not talking about Docker here, just the native install of Postgres.

If so, it would be great to have documentation on how to compile the extension library files (.dylib on MacOS I think and not sure on Windows) for these platforms.

---

Similarly, assuming Windows and Mac compilation is possible, it would be great to have zips for these in the Releases tab.

**Comment by VoVAllen** [comment, marked as answer]
We don't support MacOS and windows for now due to the inter process communication design. We rely on the shared memory API on linux now. And it's hard to build up reliable CI on github action for Mac and Windows, so we do not have plan for it also
